### PR TITLE
Document seal_name parameter to entropy

### PR DESCRIPTION
**Pull request by sgmiller**
Seal HA added this parameter to let people pick a seal to get entropy from
but it wasn't documented.

### CODE REVIEW

File: website/content/docs/configuration/entropy-augmentation.mdx [hunk 1]
```diff
@@ -59,3 +59,6 @@ These parameters apply to the `entropy` stanza in the Vault configuration file:
 - `mode` `(string: <required>)`: The mode determines which Vault operations requiring
   entropy will sample entropy from the external source. Currently, the only mode supported
   is `augmentation` which sources entropy for [Critical Security Parameters (CSPs)](/vault/docs/enterprise/entropy-augmentation#critical-security-parameters-csps).
+- `seal_name` `(string: '')`: If present, specifies which seal (by name) in a [Seal HA](/vault/docs/concepts/seal#seal-high-availability)
+  setup will be used to source entropy.  If absent, entropy will be sourced from
+  the lowest priority seal, then the next lowest if the first was unavailable, and so on.
```
Comment (schavis) [inline website/content/docs/configuration/entropy-augmentation.mdx:64]:
```suggestion
- `seal_name` `(string: '')`: Specifies which seal (by name) in a 
  [Seal HA](/vault/docs/concepts/seal#seal-high-availability) setup to use to
  source entropy. By default, Vault sources entropy in reverse priority order 
  from the available seals.
```

Style correction: write in active voice

**Review by schavis — CHANGES_REQUESTED** [review]

File: website/content/docs/configuration/entropy-augmentation.mdx (hunk 1 above)
Comment (sgmiller) [reply website/content/docs/configuration/entropy-augmentation.mdx]: Made a compromise edit to keep the high availability nature of the seal priority clear.

File: website/content/docs/configuration/entropy-augmentation.mdx [hunk 2]
```diff
@@ -59,3 +59,5 @@ These parameters apply to the `entropy` stanza in the Vault configuration file:
 - `mode` `(string: <required>)`: The mode determines which Vault operations requiring
   entropy will sample entropy from the external source. Currently, the only mode supported
   is `augmentation` which sources entropy for [Critical Security Parameters (CSPs)](/vault/docs/enterprise/entropy-augmentation#critical-security-parameters-csps).
+- `seal_name` `(string: '')`: Specifies which seal (by name) in a   [Seal HA](/vault/docs/concepts/seal#seal-high-availability) setup to use to  source entropy. 
+  By default, Vault sources entropy from the first available seal by increasing priority.
```
Comment (kubawi) [inline website/content/docs/configuration/entropy-augmentation.mdx:63]:
```suggestion
  By default, Vault sources entropy from the first available seal by increasing its priority.
```

Would this make sense? It reads better for me, but it's your call. I'm leaving an approval anyways 😄

**Review by kubawi — APPROVED** [review]
LGTM

**Review by stevendpclark — APPROVED** [review]

File: website/content/docs/configuration/entropy-augmentation.mdx (hunk 2 above)
Comment (schavis) [reply website/content/docs/configuration/entropy-augmentation.mdx:63]: Alternative suggestion to hopefully address some of the confusion:
```suggestion
  By default, Vault sources entropy from the first available seal moving from lowest to highest priority.
```

**Review by schavis — APPROVED** [review]
Added an optional suggestion to clarify some lingering confusion but otherwise lgtm

**Pull request merged**